doc/topics/tutorials/libcloud.rst

```diff
@@ -0,0 +1,239 @@
+.. _tutorial-libcloud:
+
+==============================================================================
+Using Apache Libcloud for declarative and procedural multi-cloud orchestration
+==============================================================================
+
+.. versionadded:: Oxygen
+
+.. note::
+
+    This walkthrough assumes basic knowledge of Salt and Salt States. To get up to speed, check out the
+    :ref:`Salt Walkthrough <tutorial-salt-walk-through>`.
+
+Apache Libcloud is a Python library which hides differences between different
```

Comment:
Weird formatting here. Might be good to just make a hard stop at 80 chars for this whole section of test.